modules/terraform-aci-fabric-link-level-policy/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "name" {
+  description = "Fabric link level policy name."
+  type        = string
+
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9_.-]{0,64}$", var.name))
+    error_message = "Allowed characters: `a`-`z`, `A`-`Z`, `0`-`9`, `_`, `.`, `-`. Maximum characters: 64."
+  }
+}
+
+variable "description" {
+  description = "Fabric link level policy description."
+  type        = string
+  default     = ""
+
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9\\!#$%()*,-./:;@ _{|}~?&+]{0,128}$", var.description))
+    error_message = "Allowed characters: `a`-`z`, `A`-`Z`, `0`-`9`, `\\`, `!`, `#`, `$`, `%`, `(`, `)`, `*`, `,`, `-`, `.`, `/`, `:`, `;`, `@`, ` `, `_`, `{`, `|`, }`, `~`, `?`, `&`, `+`. Maximum characters: 128."
+  }
+}
+
+variable "link_debounce_interval" {
+  description = "Link debounce interval. Minimum value: 0. Maximum value: 5000."
+  type        = number
+  default     = 0
+
+  validation {
+    condition     = var.link_debounce_interval == 0 || (var.link_debounce_interval >= 0 && var.link_debounce_interval <= 5000)
```

Comment:
Simplify `condition` to `var.link_debounce_interval >= 0 && var.link_debounce_interval <= 5000`